deploy/crds/k8s.nginx.org_nginxingresscontrollers_crd.yaml

```diff
@@ -81,6 +83,11 @@ spec:
               description: Enable TLS Passthrough on port 443. Requires enableCRDs
                 set to true.
               type: boolean
+            extraLabels:
```

Comment:
I think it's better to create a new field `service`, which will include the parameters of the service.

```
apiVersion: k8s.nginx.org/v1alpha1
 kind: NginxIngressController
 metadata:
   name: my-nginx-ingress-controller
   namespace: my-nginx-ingress
 spec:
   service: 
     extraLabels: ...
```

Otherwise, on its own `extraLabels` could be confusing - as it gives impression that those labels will be applied to everything, not only service.

I know we have `serviceType` under `spec` right now. This is something we can change when we move from `alpha1` to `v1`.

In the future, we'll extend service to have more fields:

```
apiVersion: k8s.nginx.org/v1alpha1
 kind: NginxIngressController
 metadata:
   name: my-nginx-ingress-controller
   namespace: my-nginx-ingress
 spec:
   service: 
     extraLabels: ...
     annotations: ... 
     externalTrafficPolicy: ... 
     . . .
```